package.json

```diff
@@ -37,13 +37,15 @@
   "homepage": "https://github.com/gcanti/hyper-ts",
   "devDependencies": {
     "@types/benchmark": "^1.0.31",
-    "@types/express": "4.11.0",
+    "@types/express": "^4.17.8",
```

Comment:
I had to update some type declarations because some type error was raised by `supertest`.